docs/beman-standard.md

```diff
@@ -274,3 +277,8 @@ SPDX-License-Identifier: <SPDX License Expression>
 
 **[FILE.COPYRIGHT]** RECOMMENDATION: Source code files should NOT include a
 copyright notice following the SPDX license identifier.
+
+## C++
```

Comment:
Not a single change in this section, but I think it's better to have inside C++ file conventions at the end, I expect to expand this section in the future, while the other may be stabilized soon.

<img width="234" alt="image" src="https://github.com/user-attachments/assets/aa9f4b36-c3b6-430e-a1d2-97097c7b0e89">